server/src/internalClusterTest/java/org/opensearch/cluster/shards/ClusterShardLimitIT.java

```diff
@@ -457,6 +526,29 @@ private void setShardsPerNode(int shardsPerNode) {
         }
     }
 
+    private void setIgnoreHiddenIndex(boolean ignoreHiddenIndex) {
+        try {
+            ClusterUpdateSettingsResponse response;
+            if (frequently()) {
```

Comment:
can we revisit if the usage of frequently is required here?

Reply:
yes we will check if for both persistent and transient the setting is getting applied.